### ISSUE TITLE
Sonarr not starting

### ISSUE THREAD

**Issue by Webrow** [SynoCommunity/spksrc#2413]
### Setup

_Package Name:_ Sonarr
_Package Version:_ 20150731-8

_NAS Model:_ DS214+
_NAS Architecture:_ ?
_DSM version:_ DSM 6.0.1-7393 Update 2
### Expected behavior

Start sonarr from web interface, sonarr should start after a while
### Actual behavior

Sonarr does not start
### Steps to reproduce

_1._ Clean DSM DSM 6.0.1-7393 Update 2 install, formatted drives
_2._ Install mono from community
_3._ install sonarr from community
#### Package log

_Check Package Center or `/usr/local/{package}/var/`_

```
Cant copy from screen 
installing package version ......
package install completed.......
```
#### Other logs

_E.g. `/var/log/messages` or `/var/log/synopkg.log`_

```
None with anything about mono or sonarr
```
#### Things I've tried

the mono from the community seems to be segfaulting
(reproduce, just run the installed mono on this DSM version).
So I installed the mono version from synology itself, to try sonarr with it. 
With the installation of the synology mono, I also changed the start-stop-script ( to use the bin of the synology mono instead of the comunnity mono ). 

Hope anyone can give ma a hand here, been busy for about a day already trying everything.

**Comment by KaraokeStu** [comment]
This may be related to the adduser problem we're having with packages in DSM 6

**Comment by Webrow** [comment]
is there anything I can try to find out the error? Thats a bit of the
problem, NOTHING is throwing an error. when I use the mono nzbdrone.exe
command everything starts, and a shitload get printed to stdout.

I replaced the start-stop-script with an echo, so i could see the exact
command for the start stop daemon.

excecuting that yields no output or whatever. If you want me to help with
the problem, orders are welcome.

Op 22 jul. 2016 15:53 schreef "Stuart Duncan" <notifications@github.com>:

**Comment by KaraokeStu** [comment]
Check this:

https://github.com/SynoCommunity/spksrc/issues/2398

**Comment by Webrow** [comment]
tried his change of command, his whole change, the mono package changes,
all those aint working.

Op 22 jul. 2016 16:58 schreef "Stuart Duncan" <notifications@github.com>:

**Comment by Safihre** [comment]
Sonarr and other test packages for DSM6.1 and various architectures are available here #3138. 
They also include a new way of handling permissions and should fix existing issues.
We hope to publish the packages to the official repo soon and appreciate all testing and reports on the functioning of these packages.
